docs/performance/tune-performance.md

```diff
@@ -331,6 +331,126 @@ cudaProviderOptions.UpdateOptions(providerOptionsDict);
 SessionOptions options = SessionOptions.MakeSessionOptionWithCudaProvider(cudaProviderOptions);  // Dispose this finally
 ```
 
+### Using CUDA Graphs in the CUDA EP
```

Comment:
This is going to be advertised as a preview feature in the release. Should the documentation have something to reflect that @natke ?

Reply:
I am not sure. @faxu any advice ?

Reply:
Append "(Preview)" in the heading.